---
format:
  html: default
  docx: 
    reference-doc: custom-reference-doc.docx

bibliography: references.bib
csl: asm.csl #Get themes at https://github.com/citation-style-language/styles
crossref:
  custom:
    - kind: float
      key: suppfig
      latex-env: suppfig
      reference-prefix: Supplemental Figure
      space-before-numbering: true
      latex-list-of-description: Supplementary Figure
    - kind: float
      key: supptab
      latex-env: supptab
      reference-prefix: Supplemental Table
      space-before-numbering: true
      latex-list-of-description: Supplementary Table
    - kind: float
      key: box
      latex-env: box
      reference-prefix: Box
      space-before-numbering: true
      latex-list-of-description: Box
---

```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
suppressPackageStartupMessages(library(tidyverse))
suppressPackageStartupMessages(library(knitr))

options(tidyverse.quiet = TRUE)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x, digits=2){

  if(is.list(x)){
    x <- unlist(x)
  }
  if(is.numeric(x)){
      sprintf(paste0("%0.", digits, "f"), x)
  } else {
      paste(x)
  }
}
knitr::knit_hooks$set(inline=inline_hook)
```

# A proposal for a simplified reptarenavirus taxonomy based on reassortment compatibility

<br>

Mark D. Stenglein^1^*, Buchmeier MJ, Clegg CS, Gonzalez  JP, Günther S, Hepojoki J, Kuhn JH, Okogbenin S, Radoshitzky SR, Sironi M, de la Torre JC  

(**I am hoping all *Arenaviridae* SG members would be willing to sign on to this proposal**)

<br>

*: Correspondence: Mark.Stenglein@colostate.edu

<br>

1\. Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, CO 80523
2\. Other affiliations

{{< pagebreak >}}

## Abstract

The reptarenaviruses are a large group of two-segmented snake-infecting arenaviruses. Some reptarenavirus-infected snakes develop boid inclusion body disease, a progressive neurologic condition. Other snakes remain persistently infected without manifesting clinical signs. Reptarenavirus coinfection is common, and often consists of mixtures of multiple S and L genome segments. Coinfection provides an opportunity for genome segments to reassort and recombine and both phenomena have been commonly detected. This widespread coinfection, reassortment, and recombination complicate taxonomic classification, and consequently 85% of reptarenavirus sequences are unclassified under current criteria. Here, we propose to substantially simplify reptarenavirus classification into two species, corresponding to two groups of reassorting viruses. This is analogous to the classical biological species concept based on reproductive isolation and follows the taxonomic example of other groups of frequently reassorting viruses, including influenza viruses and rotaviruses. This simpler classification scheme will result in classification of all available reptarenavirus sequences and will better accommodate the biological and genotypic complexity exhibited by this fascinating group of viruses.

- TODO: methods
- TODO: update github repo

{{< pagebreak >}}

## Introduction

Reptarenaviruses are segmented single-stranded RNA viruses in the *Arenaviridae* family. This family includes viruses that infect mammals (genus *Mammarenavirus*), snakes (genera *Reptarenavirus* and *Hartmanivirus*), fish (*Antennavirus*), and unknown hosts (*Innmovirus*) [@Radoshitzky2023; @Ly2023]. Arenaviruses have genomes of ~10.5 kb total length across 2-3 segments with negative sense or ambisense coding organization. Reptarenaviruses have 2 segments with 2 genes on each segment in opposite coding orientations. The reptarenavirus large (L) segment encodes a multifunctional polymerase (L protein) and a short Z protein. The small (S) segment encodes a glycoprotein precursor (GPC) and a nucleoprotein (NP) [@Radoshitzky2023].

Reptarenaviruses were originally identified in association with cases of inclusion body disease (IBD), a progressive neurological disease of snakes [@Schumacher1994; @Chang2010; @Stenglein2012; @Hetzel2013; @Bodewes2013]. Experimental infections confirmed a causal role in IBD pathogenesis, but not all infected snakes got sick and disease manifestation appears to be a function of multiple variables, including host species [@Stenglein2017; @Hetzel2021; @Chang2016; @Hyndman2019; @Simard2020]. Snakes can remain persistently and subclinically infected for months or years with high viral RNA levels in multiple tissues [@Stenglein2017; @Hetzel2021].

Reptarenavirus coinfection is common and has been detected in both captive and wild snakes [@Stenglein2015; @Hepojoki2015; @Keller2017; @Windbichler2019 ; @Argenta2020 ; @AlfaroAlarcn2022]. In one survey of captive reptarenavirus-infected snakes from across the USA, 27 of 48 snakes were coinfected (56%), and sampling did not enrich for coinfection [@Stenglein2015]. Coinfections consisted of complex mixtures of multiple S and L segment genotypes, with L segment genotypes typically outnumbering S segment genotypes [@Stenglein2015; @Hepojoki2015]. Prolonged subclinical infection and an apparent lack of superinfection exclusion likely contribute to the high degree of multiple infection [@Lintala2024]. 

Prolonged coinfection provides an opportunity for genome segments to compete, reassort, and recombine [@Holmes2009; @Pontremoli2018]. Indeed, reptarenavirus reassortment is common, as is recombination, which is normally an unusual feature of negative strand RNA virus biology [@Stenglein2015; @Hepojoki2015; @Holmes2009]. Coinfection, recombination, and reassortment have been detected in mammarenaviruses but appear to be substantially rarer than in reptarenavirus infection [@Weaver2000; @Charrel2001; @Fernandes2018; @Cuypers2022] . 

Current reptarenavirus species demarcation criteria were established before the extent of coinfection and reassortment was evident, and was based on assumptions that map poorly onto reptarenavirus biology. This paper documents the resulting issues and propose a simple alternative that will result in classification of all available reptarenavirus sequences and will accommodate the remarkable biological complexity of the group.

{{< pagebreak >}}

## Results

### The state of reptarenavirus classification

<!-- R objects with information about reptarenavirus classification --> 
<!-- this RDS output from script in this repo: bin/reptarenavirus_taxonomy.R -->
```{r include=F}
rept_spp <- readRDS("./RDS/reptarenavirus_species.RDS")
```

The International Committee on Taxonomy of Viruses (ICTV) oversees viral taxonomy [@Black2025]. ICTV study groups are tasked with evaluating taxonomic changes for particular groups of viruses, including the formation of new taxa [@Simmonds2025]. 

The *Arenaviridae* study group has developed classification criteria for reptarenaviruses that are similar to criteria for other genera in this family (**[Box @box-demarcation-criteria]**). The main sequence-based criterion is that viruses share less than 80% pairwise nucleotide (nt) identity between S segments and less than 76% pairwise identity in the L segment. The ICTV currently recognizes `r inline_hook(rept_spp$n_spp, 0)` reptarenavirus species: *`r inline_hook(rept_spp$spp_abbr_names)`*[@Radoshitzky2023; @ICTV_Reptarenavirus_page]. Two or more sequences (at least one one S and one L) are linked to each of these species. 

___________________
- the virus shares less than 80% nucleotide sequence identity in the S RNA segment and less than 76% identity in the L RNA segment with other viruses;
- association of the virus with a distinct main host or a group of sympatric hosts;
- dispersion of the virus in a distinct defined geographical area; and/or
- the virus shares less than 88% NP amino-acid sequence identity with other viruses @Radoshitzky2015 

**Current reptarenavirus species demarcation criteria. **From: [https://ictv.global/report/chapter/arenaviridae/arenaviridae/reptarenavirus](https://ictv.global/report/chapter/arenaviridae/arenaviridae/reptarenavirus)

::: {#box-demarcation-criteria}
:::
___________________

<!-- R objects with information about classification of NCBI reptarenavirus ssequences --> 
<!-- this RDS output from script in this repo: bin/reptarenavirus_taxonomy.R -->
```{r include = FALSE}
cc <- readRDS("./RDS/classified_values.RDS")
nn <- readRDS("./RDS/closest_MW091469_relative.RDS")
```

Only `r inline_hook(cc$fraction_classified * 100, 0)`% of reptarenavirus sequences in the National Center for Biotechnology Information (NCBI) nucleotide database are classified as belonging to a reptarenavirus species under current taxonomy. This includes `r inline_hook(cc$classified_S, 0)` of `r inline_hook(cc$S_sequences, 0)` S sequences (`r inline_hook(cc$fraction_classified_S * 100, 0)`%), and `r inline_hook(cc$classified_L, 0)` of `r inline_hook(cc$L_sequences, 0)` L sequences (`r inline_hook(cc$fraction_classified_L * 100, 0)`%; **[Fig @fig-classified-unclassified]**; **[Supp. Fig. @suppfig-classified-unclassified-L-sup]**; **[Supp. Fig. @suppfig-classified-unclassified-S-sup]**). Many unclassified sequences are not closely related to classified sequences, meaning that substantial reptarenavirus genetic diversity is not being captured by current taxonomy (**[Fig @fig-classified-unclassified]**). For instance, unclassified L segment sequence `r inline_hook(nn$this_accession)` shares only `r inline_hook(nn$pairwise_identity*100, 0)`% pairwise nt identity with the nearest classified sequence, `r inline_hook(nn$accession)` (**[Supp. Fig. @suppfig-classified-unclassified-L-sup]**). Analyses dependent on classification metadata therefore do not capture the full genetic diversity and complexity of the group. This presents an issue for computational virology research that is increasingly reliant on large datasets and associated metadata [@Simmonds2017; @Camargo2022; @Fiamenghi2025]. 

::: {#fig-classified-unclassified}
![](./figures/classified_unclassified_figure.png)

Most reptarenavirus sequences are unclassified under the current classification scheme. Midpoint rooted maximum-likelihood trees from nucleotide alignments of reptarenavirus L protein (L) and nucleoprotein (NP) coding sequences (CDS). Red colored circles denote currently classified sequences. [Supp. Fig. @suppfig-classified-unclassified-L-sup] and [Supp. Fig. @suppfig-classified-unclassified-S-sup] show these trees with tip labels and support values.
:::

### Multiple infection, reassortment, and recombination produce taxonomic uncertainty

Most reptarenavirus sequences remain unclassified because it has not been clear how to classify them in a sensible manner under current criteria. A central issue is the widespread occurrence of multiple infection. Individual snakes are often infected by reptarenavirus populations consisting of many genetically diverse S and L segments [@Stenglein2015; @Hepojoki2015; @Keller2017; @Windbichler2019 ; @Argenta2020 ; @AlfaroAlarcn2022]. These populations can be transmitted horizontally and vertically between snakes and can be isolated and propagated in tissue culture as ensembles [@Stenglein2015; @Keller2017]. Although most sampled snakes have been captive, coinfection has also been observed in wild-caught snakes [@AlfaroAlarcn2022].

The diverse sets of sequences recovered from several multiply infected snakes illustrate the taxonomic challenges posed by multiple infection (**[Fig @fig-multiply-infected]**). For instance, snake #8 from [@Hepojoki2015] was infected by a reptarenavirus population consisting of a single S segment and 4 phylogenetically diverse L segments. Snake #26 from [@Stenglein2015] yielded 2 S sequences and 4 L sequences. Snake #33 from [@Stenglein2015] was infected by a population consisting of a single S segment and 10 L segments (**[Fig @fig-multiply-infected]**).

::: {#fig-multiply-infected}
![](./figures/multiple_infection_figure.png)

Reptarenavirus multiple infection prevents straightforward classification. This co-phylogeny shows trees made from L and NP coding sequences as in [Fig @fig-classified-unclassified], with lines connecting L and S segments that were recovered from 3 representative multiply infected snakes. Different colors indicate sequences recovered from different snakes: snakes #26 and #33 from [@Stenglein2015], and snake #8 from [@Hepojoki2015]. [Supp. Fig. @suppfig-classified-unclassified-L-sup] and [Supp. Fig. @suppfig-classified-unclassified-S-sup] show these trees with tip labels and support values.
:::

The current classification criteria are based on the assumption that S and L segments co-occur as pairs in a predictable fashion. This assumption does not map well onto cases of multiple infection and leads to uncertainty. Should the 10 S-L combinations from snake #33 be assigned to 10 different reptarenavirus species? This would mean that this snake was co-infected by reptarenaviruses of 10 different species that shared a single S segment (**[Fig @fig-multiply-infected]**). Similarly, for snake #26, it's not clear which of the 4 L segments "belong to" which of the 2 S segments. Should the 8 combinations of S and L genotypes in this snake be considered 8 different species?

<!-- R objects with information about example reassortant L-S pairs -->
<!-- this RDS output from script in this repo: bin/reptarenavirus_taxonomy.R -->
```{r include = FALSE}
min_example_S_dist <- readRDS("./RDS/min_example_S_dist.RDS")
min_example_L_dist <- readRDS("./RDS/min_example_L_dist.RDS")
max_example_L_dist <- readRDS("./RDS/max_example_L_dist.RDS")
```

The different L-S combinations in multiply infected snakes represent a form of reassortment (**[Fig @fig-multiply-infected]**). Even in the absence of multiple infection reptarenavirus reassortment is common, further complicating classification, as seen in a L-NP cophylogeny from singly infected snakes (**[Fig @fig-singly-infected]**). For example, S segments KP071610, KP071539, and KP071606 are highly similar, sharing ≥ `r inline_hook(min_example_S_dist*100, 1)`% pairwise nt identity in their NP CDS. In contrast, the L segments from these 3 snakes, KP071611, KP071540, and KP071607, are divergent, sharing between `r inline_hook(min_example_L_dist*100, 1)`% and `r inline_hook(max_example_L_dist*100, 1)`% pairwise nt identity.


::: {#fig-singly-infected}
![](./figures/single_infection_figure.png)

Reptarenavirus reassortment is evident in singly infected snakes: snakes from which a single S and L segment was recovered. This co-phylogeny shows trees made from L and NP coding sequences as in [Fig @fig-classified-unclassified] and [Fig @fig-multiply-infected], with lines connecting L and S segments that were recovered from the same snake. Different colors indicate sequences recovered from [@Stenglein2015] (blue), or [@Hepojoki2015] (red). Connecting lines are only shown for sequences from these 2 studies for which suitable metadata existed. Examples of reassortant reptarenavirus genotypes, with nearly identical NP CDS but divergent L CDS are indicated with bolded connecting lines.
:::

Reassortment imposes a combinatorial classification burden. If you were to define reptarenavirus genotypes as sharing ≤ 80% pairwise nt identity ([@box-demarcation-criteria]), then there would be 13 NP genotypes and 31 L genotypes. These have the potential to reassort to produce 403 L-S combinations, each of which would require the establishment of a new reptarenavirus species under current criteria. Different species would share identical or nearly identical S or L segments. Intrasegmental recombination [@Stenglein2015] makes this problem even worse by increasing the number of possible S and L segment combinations.

### There is a simple solution 

Fortunately, there is a simple solution, one that has been adopted for other groups of frequently reassorting viruses. That is to lump groups of reassorting viruses into single species. In this new scheme, all known reptarenavirus sequences would be reclassified into one of two existing species (**[Fig @fig-new-species]**). Most reptarenavirus sequences would be assigned to the species *Reptarenavirus giessenae*, with exemplar isolate University of Giessen virus 1 (UGV1; L: KR870022; S: KR870012). Eight sequences, all from annulated tree boas (*Corallus annulatus*), would be classified as *Reptarenavirus californae*, with exemplar isolate CAS virus (CASV; L: JQ717261; S: JQ717262). The species *R. aurei*, *R. commune*, and *R. rotterdamense* would be subsumed by *R. giessenae* and eliminated.  [@supptab-acc-spp-map] lists all available reptarenavirus sequence accessions and their proposed species assignments. 

::: {#fig-new-species}
![](./figures/new_species_figure.png)

Proposed new reptarenavirus species. All available reptarenavirus species would be reclassified into two species: *R. giessenae* (blue) or *R. californae* (orange). This co-phylogeny shows trees made from L and NP coding sequences as in [Fig @fig-classified-unclassified], with lines connecting L and S segments that were recovered from the same snakes from [@Stenglein2015] and [@Hepojoki2015]. Connecting lines are only shown for sequences from these 2 studies for which suitable metadata existed. 
:::

These two new proposed species correspond to monophyletic clades of viruses separated by long branches in L and NP trees (**[Fig @fig-new-species]**). These species share two properties that form the basis for the proposed new classification (**[@box-proposed-new-demarcation-criteria]**). First, they consist of groups of reassorting viruses (**[Fig @fig-new-species]**). (Reassortment has not been observed for CASV, though only several CASV-like viruses have been characterized). Second, L and NP protein sequences share ≥ 60% pairwise amino acid identity within species and ≤ 60% between species (**[Fig @fig-pairwise-distances]**). 

___________________
- Reptarenavirus species consist of groups of reassorting viruses.
- L and NP protein sequences from different reptarenavirus species share ≤ 60% pairwise amino acid identity. 

**Proposed new reptarenavirus species demarcation criteria.**

::: {#box-proposed-new-demarcation-criteria}
:::
___________________

::: {#fig-pairwise-distances}
![](./figures/pairwise_distance_figure.png)

Pairwise identities of (A) L and (B) NP protein sequences within and between proposed new reptarenavirus species. The pairwise identity of L and NP protein sequences from within-species comparisons and between-species comparisons was determined from multiple sequence alignments and plotted as histograms. Species designations here are as in [Fig @fig-new-species].
:::

{{< pagebreak >}}



## Discussion

Taxonomists are famously categorized as "lumpers" or "splitters": those that wish to lump more taxa into larger groups vs. those that wish to split taxa into ever finer groups [@Endersby2009]. Here we are proposing substantial lumping, but this is driven by necessity. Current reptarenavirus taxonomy is rooted in assumptions that do not apply. When classification criteria were initially devised, the extent of multiple infection and reassortment was simply not yet known. Nevertheless, this has resulted in an inability to classify most reptarenavirus sequences. The proposed reclassification solves this issue and should be able to accommodate future expansions in known reptarenavirus diversity. 

It is possible, of course, that recovery of additional reptarenavirus sequences could challenge the basis for this reclassification. For instance, additional sampling could reveal viruses with CASV-like S segments and UGV1-like L segments (**[Fig @fig-new-species]**). In that case, reptarenavirus classification could simply be revised again. 

This new scheme is based on the successful precedents of influenza viruses, rotaviruses, and bluetongue viruses, all of which reassort and contain substantial genetic diversity within single species. The 1.48 million influenza A virus sequences in the NCBI nucleotide database all belong to the species *Alphainfluenzavirus influenzae*. This large-scale lumping does not impede research or clinical practice. Influenza (A) virus research, as an example, seems unencumbered by the single-species designation. This may be because taxonomy is largely an academic pursuit. It is useful only insofar as it improves understanding of underlying biology, leads to practical application, or serves as meaningful metadata. 

Subspecies classification frameworks, such as influenza virus hemagglutinin/neuraminidase subtyping, have been developed for all of these groups and this proposed taxonomic reorganization does not preclude future reptarenavirus subspecies categorization. Moreover, viral species are not equivalent to virus names, and new species designations do not mean that existing virus names need to change [@Zerbini2025].

Here, we are inferring reassortment compatibility based on cooccurence of sequences in individual infections. The presence of ten L segments and one S segment present in snake #33 (**[Fig @fig-multiply-infected]**), for instance, provides evidence that all ten L segments are genetically compatibly with the single S segment. This is not the same as experimental demonstration of genetic compatibility. But as virus sequences are identified through metagenomics at an ever increasing rate, it will not be possible to develop experimental systems for all groups of viruses [@Greninger2018]. So, well-supported inference based on sequence data will be increasingly useful. 

Reassortment is a form of gene flow, and classification based on reassortment compatibility is analogous to Mayr's biological species concept, which defines species as "groups of actually or potentially interbreeding natural populations, which are reproductively isolated from other such groups" [@Mayr1942]. The connection between the proposed reptarenavirus classification and this classic definition is appealing, as is the new alignment with other groups of frequently reassorting viruses. Viral taxonomists might consider whether other groups of viruses that exhibit frequent genetic exchange are due for similar taxonomic revision. 

{{< pagebreak >}}

## Methods

We downloaded all coding-complete reptarenavirus sequences in the NCBI nucleotide database on Oct 29, 2025. We parsed the taxonomic classification of each sequence from the NCBI records. We extracted coding sequences for each gene and created multiple sequence alignments for each gene using coding sequences and translated protein sequences using the MAFFT aligner v7.525 [@Katoh2013]. We manually inspected alignments to confirm the absence of any obvious alignment artifacts using Geneious software v2025.1.2 (https://www.geneious.com).  We used iqtree v3.0.1 to create maximum likelihood phylogenetic trees from each multiple sequence alignment, allowing iqtree to select the best-fitting model by specifying parameter -m MFP [@Minh2020; @Kalyaanamoorthy2017].  Branch support values were determined using UltraFast bootstrapping and SH-like approximate likelihood ratio testing (SH-aLRT) [@Hoang2017]. Non-tanglegram trees were visualized using the ggtree R package [@Yu2016].  Cophylogenies (tanglegrams) were created using the phytools R package [@Revell2024].  Figures were generally output from R scripts as PDFs and in some cases further annotated using Affinity Designer.

Sequence and virus metadata was obtained from several sources.  Some metadata was parsed directly from genbank format sequence files downloaded from NCBI (accession, organism name, and sequence taxonomic ID (taxid)) using Biopython functions [@Cock2009]. Coinfecting sequences were mapped to individual snakes from Table 1 in [@Hepojoki2015]. Reptarenavirus species information was parsed from the ICTV master species list (MLS 40.v2.20251013) obtained from the ICTV website. 

This paper is implemented as a fully reproducible workflow based on make, nextflow v25.10.0, R, and singularity-ce version 4.2.1 [@DiTommaso2017; @Wickham2019; @RCoreTeam2025]. Code and data for this paper are available at: https://github.com/stenglein-lab/reptarenavirus_taxonomy.

{{< pagebreak >}}

## Acknowledgements

Research on reassortment in the Stenglein lab is supported by NSF IOS 2048214 and NIH R01 AI177711.	

## Data availability

This paper is implemented as a fully reproducible workflow based on make, nextflow, and singularity. Code and data for this paper are available at: https://github.com/stenglein-lab/reptarenavirus_taxonomy.

{{< pagebreak >}}

## References

::: {#refs}
:::

{{< pagebreak >}}

### Supplemental Figures

::: {#suppfig-classified-unclassified-L-sup}

![](./figures/supplemental/Supplemental_figure_X_L_classified_unclassified_tree.png)

Most reptarenavirus L sequences are unclassified under the current classification scheme. Midpoint rooted maximum-likelihood trees from nucleotide alignments of reptarenavirus L protein (L) coding sequences. Red colored circles denote currently classified sequences.
::: 

::: {#suppfig-classified-unclassified-S-sup}

![](./figures/supplemental/Supplemental_figure_X_S_classified_unclassified_tree.png)

Most reptarenavirus NP sequences are unclassified under the current classification scheme. Midpoint rooted maximum-likelihood trees from nucleotide alignments of reptarenavirus nucleoprotein (NP) coding sequences. Red colored circles denote currently classified sequences.
::: 

::: {#supptab-acc-spp-map}

[Link to table](./tables/Supplemental_table_accession_proposed_species_map.txt){text-align="left"}

A map of reptarenavirus sequence accessions to proposed new species.
::: 
