---
csl: arch_virol.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 12pt
output:
  bookdown::html_document2: 
    number_sections: false 
    global_numbering: true 
  bookdown::pdf_document2: 
    number_sections: false 
    global_numbering: true 
  bookdown::word_document2: 
    number_sections: false 
    global_numbering: true 
    includes:
      in_header: header.tex
    keep_tex: yes
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
library(knitr)
library(readr)
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x, digits=2){

  if(is.list(x)){
    x <- unlist(x)
  }
  if(is.numeric(x)){
      paste(format(x,big.mark=',', digits=digits, nsmall=digits, scientific=FALSE))
  } else {
      paste(x)
  }
}
knitr::knit_hooks$set(inline=inline_hook)
```


\pagenumbering{arabic}
\linenumbers
\doublespacing

# A proposal to taxonomically classify reptarenaviruses into two species based on reassortment compatibility

\vspace{20mm}

**Mark D. Stenglein^1$\dagger$^, List: members of the ICTV arenaviridae sg, other co-authors**

\vspace{40mm}

Affiliations

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: mark.stenglein@colostate.edu

\vspace{20mm}

1\. Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, CO 80523
2\. Other affiliations

\vspace{30mm}

\newpage 

## Abstract

The reptarenaviruses are a large group of two-segmented snake-infecting arenaviruses. Reptarenavirus infection has been linked to boid inclusion body disease, although snakes can remain persistently infected without manifesting clinical signs. Reptarenavirus coinfection is common, and often consisted of complex mixtures of multiple S and L genome segments. Coinfection provides an opportunity for genome segments to reassort and recombine and both have been detected in reptarenaviruses. This coinfection, reassortment, and recombination complicate taxonomic classification, and most reptarenavirus sequences are unclassified under current criteria. Here, we propose to taxonomically reclassify the reptarenaviruses into two species, corresponding to two groups of reassorting viruses.  This is analogous to the classical biological species concept based on reproductive isolation and follows the taxonomic example of other groups of frequently reassorting viruses, including influenza viruses and rotaviruses.  This simpler classification scheme will result in classification of all available reptarenavirus sequences and will better accommodate the biological complexity exhibited by this fascinating group of viruses.

## Introduction

Reptarenaviruses are segmented ambisense single-stranded RNA viruses in the Arenaviridae family.  This family includes viruses that infect mammals (genus *Mammarenavirus*), snakes (genera *Reptarenavirus* and *Hartmanivirus*), and fish (*Antennavirus*) [@Radoshitzky2023; @Ly2023].  Viruses in this group have genomes of ~10.5 kb total length across 2-3 segments with negative sense or ambisense coding organization.

Reptarenaviruses were originally identified in association with cases of inclusion body disease (IBD), a progressive neurological disease of snakes [@Schumacher1994; @Chang2010; @Stenglein2012; @Hetzel2013; @Bodewes2013].  Experimental infections confirmed a causal role in IBD pathogenesis, but not all infected snakes got sick and disease manifestation appears to be a function of multiple variables, including host species [@Stenglein2017; @Hetzel2021; @Chang2016; @Hyndman2019; @Simard2020].  Snakes can remain persistently infected for months or years with high viral levels in multiple tissues [@Stenglein2017; @Hetzel2021].

Reptarenavirus coinfection is common and has been detected in both captive and wild snakes [@Stenglein2015; @Hepojoki2015; @Keller2017; @Windbichler2019 ; @Argenta2020 ; @AlfaroAlarcn2022].  In one survey 27 of 48 reptarenavirus infected snakes were coinfected (56%; snakes were not sampled to enrich for coinfection) [@Stenglein2015]. Coinfections consisted of complex mixtures of multiple S and L segment genotypes, with L segment genotypes typically outnumbering S segment genotypes [@Stenglein2015; @Hepojoki2015].  These populations can be transmitted as ensembles horizontally and vertically [@Stenglein2015; @Keller2017]. Prolonged subclinical infection and an apparent lack of superinfection exclusion likely contribute to the high degree of coinfection [@Lintala2024].   

This prolonged coinfection  provides an opportunity for the coinfecting segments to compete, reassort, and recombine [@Holmes2009].  Indeed, reptarenavirus reassortment is common, as is recombination,  which is normally an unusual feature of negative strand RNA virus biology [@Stenglein2015; @Hepojoki2015].  Coinfection, recombination, and reassortment have been detected in mammarenaviruses but appear to be substantially rarer than in reptarenavirus infection [@Weaver2000; @Charrel2001; @Fernandes2018; @Cuypers2022] . 

The high degree of complex coinfection, reassortment, and recombination complicates the taxonomic classification of reptarenavirus sequences. 

Currently, reptarenaviruses are classified according to several criteria that are similar to the classification criteria for other arenaviruses (Box/Table 1).  The main criterion is that viruses share less than 80% pairwise nucleotide sequence identity between S segments and less than 76% pairwise identity in the L segment.  This criterion would work well if infections corresponded to single S and single L segments, which they do not.  The current criteria do not consider how to classify sequences present in multiple segment coinfections, nor do they describe classification of recombinant or reassortant sequences. In other words, the current classification scheme makes assumptions that do not match the reality of reptarenavirus infection and reptarenavirus classification does not map well onto reptarenavirus biology and consequently most reptarenavirus sequences remain unclassified.  

**Box/Table 1: Current Reptareanvirus Classification Criteria**
The parameters used to assign viruses to new species in the reptarenavirus genus are:

- the virus shares less than 80% nucleotide sequence identity in the S RNA segment and less than 76% identity in the L RNA segment with other viruses;
- association of the virus with a distinct main host or a group of sympatric hosts;
- dispersion of the virus in a distinct defined geographical area; and/oror
- the virus shares less than 88% NP amino-acid sequence identity with other viruses @Radoshitzky2015 

*From: [https://ictv.global/report/chapter/arenaviridae/arenaviridae/reptarenavirus](https://ictv.global/report/chapter/arenaviridae/arenaviridae/reptarenavirus)*


Analyses that only consider classified reptarenavirus sequences therefore fail to capture the full complexity of the group. 

This paper documents the issues that arise from mapping these classification criteria onto observed reptarenavirus complexity and propose a simple solution based on the strategy used for other groups of frequently-reassorting RNA viruses, including infleunza A viruses and rotaviruses.  This proposed new scheme will result in classification of all available reptarenavirus sequences and will accomdate the remarkable genetic complexity of the group.


## Results


Pervasive reassortment has broken the link between individual S segments and L segments.  So classification schemes based only on one segment will not adequately reflect the diverse set of associated genotypes. 


There is a clear and simple solution to this problem: one that has been adopted for other groups of viruses that frequently reassort. 

Other groups of viruses that frequently reassort offer a solution to this taxonomic conundrum.  These include influenza A viruses, rotaviruses, and bluetongue viruses.  Each of these groups is lumped into a single species: Influenzavirus A, X, Y.  

Intrasegment recombination further complicates reptarenavirus taxonomy.

Enumerate and describe problems with current classification scheme, using figures 




### The state of reptarenavirus classification

<!-- R objects with information about reptarenavirus classification   --> 
<!-- this RDS output from script in this repo: bin/parse_ICTV_MLS.R -->
```{r include=F}
rept_spp <- readRDS("../RDS/reptarenavirus_species.RDS")
```

The ICTV currently recognizes `r inline_hook(rept_spp$n_spp)` reptarenavirus species: *`r inline_hook(rept_spp$spp_names)`*.  These species belong to the realm *Riboviria*, kingdom *Orthornavirae*, phylum *Negarnaviricota*, subphylum *Polyploviricotina*, class	*Bunyaviricetes*, order *Hareavirales*, family *Arenaviridae*, and genus *Reptarenavirus*.

<!-- R objects with information about classification of NCBI reptarenavirus ssequences  --> 
<!-- from: bin/reptarenavirus_distances.R -->
<!-- and : bin/plot_reptarenavirus_taxonomy.R -->
```{r include = FALSE}
cc <- readRDS("../RDS/classified_values.RDS")
nn <- readRDS("../RDS/closest_MW091469_relative.RDS")
```

Only `r inline_hook(cc$fraction_classified * 100, 1)`% of reptarenavirus sequences in the NCBI nucleotide database are classified under current taxonomy. This includes `r inline_hook(cc$fraction_classified_S * 100, 1)`% of S segment sequences and  `r inline_hook(cc$fraction_classified_L * 100, 1)`%  of L segment sequences ((`r inline_hook(cc$classified_S, 1)` of `r inline_hook(cc$S_sequences, 1)` S sequences, and  `r inline_hook(cc$classified_L, 1)` of `r inline_hook(cc$L_sequences, 1)` L sequences (Fig.  \@ref(fig:classified-unclassified-fig))).  Many unclassified sequences are not closely related to classified sequences, meaning that substantial reptarenavirus genetic diversity is not being captured by current taxonomy (Fig.  \@ref(fig:classified-unclassified-fig)).   For instance, unclassified L segment sequence `r inline_hook(nn$this_accession)` only shares `r inline_hook(nn$pairwise_identity*100, 1)`% pairwise identity with the nearest classified sequence, `r inline_hook(nn$accession)`.  Analyses that only consider classified reptarenavirus sequences therefore do not capture the full genetic diversity and complexity of the group [TODO: insert ref]. 

```{r classified-unclassified-fig, fig.cap = "Most reptarenavirus sequences are unclassified under the current classification scheme. Midpoint rooted maximum-likelihood trees from nucleotide alignments of reptarenavirus L protein (L) and nucleoprotein (NP) coding sequences. Red colored circles denote currently classified sequences. "}
fig <- readRDS("../figures/classified_unclassified_figure.RDS")
fig
```

### Issues resulting from the current classification scheme

Not only are many reptarenavirus sequences ununclassified, many are unclassifiable according to the current classification criteria.

### A large number of species would be required to handle the large number of S-L combinations

### Sequences from the same species are not monophyletic

A sequence-based classification scheme should produce monphyletic taxa. In other words, sequences from the same species should form a clade in evolutionary trees with a common ancestor that is not shared by other species.  Current reptarenavirus classification schemes result in paraphyletic species in one or both segments. 

### A large number of species would be required to handle the large number of S-L combinations

### Reassortment breaks the link between individual L and S segments

Reassortment breaks the genetic linkage between particular S and L segments. This means that viruses that share identical or nearly identical S segments but have different L segments would be classified in different species (and vice-versa), and new species would be required for each new segment combination.  This would require the establishment of Z new species beyond the current Z, and it is likely that much reptarenavirus diversity remains unsampled.



### It is not clear how to taxonomically handle sequences present in multiple coinfections


### It is not clear how recombinant sequences should be classified

The current classification scheme has potentially conflicting 


One issue with the current classificaiton


Trees: Reptarenavirus diversity and classification

NP gene trees
L gene tree
Z gene tree
GP gene tree

Tips colored/labeled by current classification. 

Examples of situations where kkkkkk


### There is a simple solution to these problems


All influenza A viruses are classified as belonging to the species Alphainfluenzavirus influenzae.  


Trees showing classification of reptarenavirus


### Proposed new scheme

Proposed new classification scheme, including species name, type species, and new classification criteria.



## Methods

This paper is implemented as a fully reproducible workflow based on make, nextflow, and singularity. Code and data for this paper are available at: https://github.com/stenglein-lab/reptarenavirus_taxonomy.

### Reptarenavirus sequences 

We downloaded all coding-complete reptarenavirus sequences in the NCBI nucleotide database on Oct 29, 2025.  We parsed the taxonomic classification of each sequence from the NCBI records.  We extracted coding sequences for each gene and created multiple sequence alignments for each gene using the MAFFT aligner vX.Y.Z.  We manually inspected alignments to confirm the absence of any obvious alignment artifacts.  We used the GARD tool implemented on the datamonkey webserver to confirm the absence of recombination within invividual gene alignments (recombination breakpoints have been detected between but not within genes).

Mapping sequences to individual infections

### Trees and tanglegrams

We created trees from individual gene multiple sequences using iqtree version 2.X using default parameters and used ggTree to visualize trees.    We used phytools to create tanglegrams.



## Discussion

Taxonomists are famously categorized as being lumpers or splitters: those that wish to create larger groups with more taxa vs. those that wish to divide taxa into ever finer groups. This is a lumping scheme, that will merge reptarenavirus sequences into a small number of new species. This 

Reptarenavirus taxonomy is rooted in assumptions about how viruses ought to behave that do not apply.  For instance, that S and L segments are neatly found together in single infections.

This classification scheme is based on the successful precedent of influenza viruses, rotaviruses, and bluetongue viruses. Subspecies classification frameworks have been developed for all of these groups. For instance, HA/NA subtyping for influenzaviruses, and XYZ for rotativruses. A similar subspecies categorization scheme could be developed for the reptarenaviruses. And new species designations do not mean that existing virus names need to be jettisoned [@Zerbini2025].


Should the classification of other groups of frequently reassorting viruses be similarly simplified?  If so, which groups?  The mammarenaviruses and antennaviruses seem to exhibit much less reassortment.  One potential candidate is the orthobunyaviruses.  Drosophila-infecting partitiviruses?  Hartmaniviruses?  Perhaps groups of viruses that frequently recombine, like viruses within the sarbecovirus genus could also.

An appealing feature of this classification scheme is its similarity to widely used species definitions for organisms in general. 

The classification scheme proposed here, and the one used by influenza viruses, etc, is analogous to Mayr's biological species concept, which defines a species as 
"groups of actually or potentially interbreeding natural populations, which are reproductively isolated from other such groups." (

Like sexual reproduction, viral reassortment and recombination represent forms of interbreeding that allow genetic mixing.

Similar also to the genetic species concept as elaborated by and others: "a group of genetically compatible interbreeding natural populations that is genetically isolated from other such groups" [https://pmc.ncbi.nlm.nih.gov/articles/PMC2771874/]

https://en.wikipedia.org/wiki/Species




## Acknowledgements

Research on reassortment in the Stenglein lab is supported by NSF X and NIH Y.   Feedback from / authorship by Jens Kuhn?

[@Paradis2018]

## References

<div id="refs"></div>

## Figure Legends

