---
csl: arch_virol.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 12pt
output:
  bookdown::html_document2: 
    number_sections: false 
    global_numbering: true 
  bookdown::pdf_document2: 
    number_sections: false 
    global_numbering: true 
  bookdown::word_document2: 
    number_sections: false 
    global_numbering: true 
    includes:
      in_header: header.tex
    keep_tex: yes
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
library(knitr)
library(readr)
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```


\pagenumbering{arabic}
\linenumbers
\doublespacing

# A proposal to taxonomically classify reptarenaviruses into two species based on reassortment compatibility

\vspace{30mm}

Outline for reptarenavirus taxonomy paper/proposal

Authors: Mark Stenglein and the ICTV arenavirus taxonomy study group

Affiliations:

\newpage 


## Abstract

The reptarenaviruses are a large group of bisegmented snake-infecting viruses. Reptarenavirus infection is common in captive snake populations and has been linked to boid inclusion body disease.  Although infection is associated with disease in some instances, snakes can be infected for long periods without manifesting clinical signs. It is common for snakes to be co-infected by reptarenaviruses.  These co-infections are often not simple co-infection by two viruses but rather infection by complex mixtures of multiple S and L genome segments. These mixed viral populations can be transmitted from snake to snake and have been detected in both captive and wild animals. Coinfection can produce Reassortment and recombination and both have been commonly detected among reptarenaviruses.  This genotypic complexity presents a problem for taxonomic classification, and the current classification scheme does not coherently map onto the observed biological complexity and has left most reptarenavirus sequences unclassified (and unclassifiable?). Here, we propose to taxonomically reclassify the reptarenaviruses into two species, corresponding to two groups of reassorting viruses.  This is analogous to the classical species definition based on reproductive isolation and follows the taxonomic example of other groups of frequently reassorting viruses, including influena viruses and rotaviruses.  This simpler classification scheme better reflects the biological complexity exhibited by this fascinating group of viruses.


## Introduction

Reptarenaviruses are bisegmented ambisense single stranded RNA viruses in the Arenaviridae family.  This family includes viruses that infect mammals (genus Mammarenavirus), snakes (Reptarenavirus), and fish (Antennavirus).  Viruses in this group share a bisegmented ambisense genome organization with L and S segments with one or more genes per segment.

Reptarenaviruses were originally identified in association with cases of inclusion body disease, a progressive neurodegenerative disease of snakes.  Subsequent experimental infections confirmed a causal role in IBD pathogenesis, but not all infected snakes get sick and disease manifestation appears to be a function of multiple variables, including host species.

Reptarenavirus co-infection is common and has been detected in both captive and wild-living snakes [@Argenta2020 ; @AlfaroAlarcn2022].  For instance, one study that recovered viral genomes from X infected snakes found that Y of them were co-infected.  A study of X wild snakes found Y.  Co-infection may result from serial infection by different strains or by transmission of a mixed population of multiple virus genotypes or both (have both been documented)?.  Superinfection exclusion.   Snakes remain infected for long periods without clearning the virus: experimentally infected boa constrictors remained infected for two years with high viral RNA levels in blood and tissues.  This prolonged coinfection  provides an opportunity for the coinfecting segments to compete, reassort, and recombine.  Reassortment is consequently common and recombination has been detected, which is normally an unusual feature of negative strand RNA virus biology.   Coinfection and reassortment have been detected in mammarenaviruses, but seems to be the exception rather than the rule, as it is in reptarenaviruses.

Currently, reptarenaviruses are classified according to several criteria that are similar to the classification criteria for other arenaviruses (Box/Table 1).  The main criterion is that viruses share less than 80% pairwise nucleotide sequence identity between S segments and less than 76% pairwise identity in the L segment.  This criterion would work if infections corresponded to single S and single L segments.  But infections commonly consist of one or several S and L segments. The current criteria do not describe how to classify sequences in these kinds of multiuple infections.
This has resulted in a classification scheme that does not map well onto the viral genotypes present in individual ainfected snakes. For instance, individual snakes can be infected by 1 S segment and Y L segments.  Currently only X of Y arenavirus sequences in the NCBI nucleotide database are classified.  Anlaytic workflows that only consider classified reptarenavirus sequences therefore fail to capture the full complexity of the group. 

from: https://ictv.global/report/chapter/arenaviridae/arenaviridae/reptarenavirus
The parameters used to assign viruses to new species in the genus are:

- the virus shares less than 80% nucleotide sequence identity in the S RNA segment and less than 76% identity in the L RNA segment with other viruses;
- association of the virus with a distinct main host or a group of sympatric hosts;
- dispersion of the virus in a distinct defined geographical area; and/oror
- the virus shares less than 88% NP amino-acid sequence identity with other viruses (Radoshitzky et al., 2015).

Pervasive reassortment has broken the link between individual S segments and L segments.  So classification schemes based only on one segment will not adequately reflect the diverse set of associated genotypes. 


There is a clear and simple solution to this problem: one that has been adopted for other groups of viruses that frequently reassort. 

Other groups of viruses that frequently reassort offer a solution to this taxonomic conundrum.  These include influenza A viruses, rotaviruses, and bluetongue viruses.  Each of these groups is lumped into a single species: Influenzavirus A, X, Y.  

Intrasegment recombination further complicates reptarenavirus taxonomy.

Enumerate and describe problems with current classification scheme, using figures 

## Results



### The state of reptarenavirus classification

The 

Under the current classification scheme, many reptarenavirus sequences are unclassified, and, as described below, unclassifiable.   X of Y reptarenavirus sequences in the NCBI nucleotide database are classified into Z species.  This leaves X sequences (X%) unclassified.  The currently classified sequences only capture a portion [can quantify using branch lenghts?] of reptarenavirus genetic diversity, as measured using .

### Issues resulting from the current classification scheme

### A majority of reptarenavirus sequences remain unclassified

Under the current scheme, most reptarenavirus sequences remain unclassified.  X of Y L segment sequences and Z of W S segment sequences are currently unclassified (Fig.  \@ref(fig:classified-unclassified-fig)).  Some unclassified sequences are not closely related to classified sequences, meaning that important reptarenavirus geneitc diversity is not being captured.  For instance, sequence X, is unclassifed and only shares Y% 




```{r classified-unclassified-fig, fig.cap = "Most reptarenavirus sequences are unclassified under the current classification scheme.  Midpoint rooted maximum-likelihood trees from nucleotide alignments of reptarenavirus L and NP coding sequences.  Red colored tips indicate unclassified sequence. "}
fig <- readRDS("figures/classified_unclassified_figure.RDS")
fig
```

### A large number of species would be required to handle the large number of S-L combinations

### Sequences from the same species are not monophyletic

A sequence-based classification scheme should produce monphyletic taxa. In other words, sequences from the same species should form a clade in evolutionary trees with a common ancestor that is not shared by other species.  Current reptarenavirus classification schemes result in paraphyletic species in one or both segments. 

### A large number of species would be required to handle the large number of S-L combinations

### Reassortment breaks the link between individual L and S segments

Reassortment breaks the genetic linkage between particular S and L segments. This means that viruses that share identical or nearly identical S segments but have different L segments would be classified in different species (and vice-versa), and new species would be required for each new segment combination.  This would require the establishment of Z new species beyond the current Z, and it is likely that much reptarenavirus diversity remains unsampled.


### It is not clear how to taxonomically handle sequences present in multiple coinfections


### It is not clear how recombinant sequences should be classified

The current classification scheme has potentially conflicting 


One issue with the current classificaiton


Trees: Reptarenavirus diversity and classification

NP gene trees
L gene tree
Z gene tree
GP gene tree

Tips colored/labeled by current classification. 

Examples of situations where kkkkkk


### There is a simple solution to these problems

Trees showing classification of reptarenavirus


### Proposed new scheme

Proposed new classification scheme, including species name, type species, and new classification criteria.



## Methods

This paper is implemented as a fully reproducible workflow based on make, nextflow, and singularity. Code and data for this paper are available at: https://github.com/stenglein-lab/reptarenavirus_taxonomy.

### Reptarenavirus sequences 

We downloaded all coding-complete reptarenavirus sequences in the NCBI nucleotide database on Oct 29, 2025.  We parsed the taxonomic classification of each sequence from the NCBI records.  We extracted coding sequences for each gene and created multiple sequence alignments for each gene using the MAFFT aligner vX.Y.Z.  We manually inspected alignments to confirm the absence of any obvious alignment artifacts.  We used the GARD tool implemented on the datamonkey webserver to confirm the absence of recombination within invividual gene alignments (recombination breakpoints have been detected between but not within genes).

Mapping sequences to individual infections

### Trees and tanglegrams

We created trees from individual gene multiple sequences using iqtree version 2.X using default parameters and used ggTree to visualize trees.    We used phytools to create tanglegrams.



## Discussion

Reptarenavirus taxonomy is rooted in assumptions about how viruses ought to behave that do not apply.  For instance, that S and L segments are neatly found together in single infections.

This classification scheme is based on the successful precedent of influnza viruses, rotaviruses, and bluetongue virues.  Subspecies classification frameworks have been developed for all of these groups.  For instance, HA/NA subtyping for influenzaviruses, and XYZ for rotativruses. A similar subspecies categorization scheme could be developed for the reptarenaviruses.  

Should the classification of other groups of frequently reassorting viruses be similarly simplified?  If so, which groups?  The mammarenaviruses and antennaviruses seem to exhibit much less reassortment.  One potential candidate is the orthobunyaviruses.  Drosophila-infecting partitiviruses?  Hartmaniviruses?  Perhaps groups of viruses that frequently recombine, like viruses within the sarbecovirus genus could also.

An appealing feature of this classification scheme is its similarity to widely used species definitions for organisms in general. 

The classification scheme proposed here, and the one used by influenza viruses, etc, is analogous to Mayr's biological species concept, which defines a species as 
"groups of actually or potentially interbreeding natural populations, which are reproductively isolated from other such groups." (

Like sexual reproduction, viral reassortment and recombination represent forms of interbreeding that allow genetic mixing.

Similar also to the genetic species concept as elaborated by and others: "a group of genetically compatible interbreeding natural populations that is genetically isolated from other such groups" [https://pmc.ncbi.nlm.nih.gov/articles/PMC2771874/]

https://en.wikipedia.org/wiki/Species




## Acknowledgements

Research on reassortment in the Stenglein lab is supported by NSF X and NIH Y.   Feedback from / authorship by Jens Kuhn?

## References

<div id="refs"></div>

## Figure Legends

